javascript/powercommands/task_provider.js

```diff
@@ -0,0 +1,23 @@
+const getTaskRefs = async (client, projectId) => {
+  console.log('Retrieving all task references for project', projectId);
+  return await client.projects.tasks(projectId);
```

Comment:
I believe "await" here is unnecessary, as this request is returning a promise, and is currently being bound in a promise which is not doing any additional logic.